testsupport/signup_request.go

```diff
@@ -204,7 +211,19 @@ func (r *signupRequest) Execute() SignupRequest {
 			}
 		}
 
-		userSignup, err = hostAwait.UpdateUserSignupSpec(userSignup.Name, doUpdate)
+		userSignup, err = hostAwait.UpdateUserSignup(userSignup.Name, doUpdate)
+		require.NoError(r.t, err)
+	}
+
+	if r.lastCluster != nil {
```

Comment:
I'm a bit confused... Why do we care about the last cluster in the request? The last cluster is fully internal thing which sets an internal preference. It should not be translated to the usersignup.spec.target cluster which is set by "clients" and a must (not a preference).